src/geode/mesh/helpers/detail/solid_merger.cpp

```diff
@@ -205,37 +198,7 @@ namespace geode
                         }
                     }
                 }
-            }
-
-            void create_adjacencies( SolidMeshMerger< dimension >& merger )
-            {
-                absl::FixedArray< bool > visited_polyhedra(
-                    merger.mesh().nb_polyhedra(), false );
-                const auto& meshes = merger.meshes();
-                for( const auto s : Indices{ merger.meshes() } )
-                {
-                    const auto& solid = meshes[s].get();
-                    for( const auto p : Range{ solid.nb_polyhedra() } )
-                    {
-                        const auto new_id = new_id_[s][p];
-                        if( visited_polyhedra[new_id] )
-                        {
-                            continue;
-                        }
-                        visited_polyhedra[new_id] = true;
-                        for( const auto f :
-                            LRange{ solid.nb_polyhedron_facets( p ) } )
-                        {
-                            if( const auto adj =
-                                    solid.polyhedron_adjacent( { p, f } ) )
-                            {
-                                const auto new_adj_id = new_id_[s][adj.value()];
-                                merger.builder().set_polyhedron_adjacent(
-                                    { new_id, f }, new_adj_id );
-                            }
-                        }
-                    }
-                }
+                merger.builder().compute_polyhedron_adjacencies();
```

Comment:
This means adjacencies are put back in input meshes, it changes the in/out assumptions, is this wanted ?

Reply:
A fix is in progress